thing-url-adapter.js

```diff
@@ -481,6 +481,15 @@ class ThingURLAdapter extends Adapter {
     super(addonManager, manifest.id, manifest.id);
     addonManager.addAdapter(this);
     this.knownUrls = {};
+
+    this.loadPoolInterval();
+  }
+
+  async loadPoolInterval() {
```

Comment:
Instead of rereading the config from the database, you can just set the poll interval on the adapter inside `loadThingURLAdapter()`.